test/integration/crud/find.test.js

```diff
@@ -781,126 +781,6 @@ describe('Find', function () {
     }
   });
 
-  /**
-   * Test findOneAndUpdate a document
-   */
-  it('shouldCorrectlyfindOneAndUpdateDocument', {
```

Comment:
The cases in this giant test case are covered in individual findAndModify tests as well as spec tests.